llm-translator.html

```diff
@@ -0,0 +1,276 @@
+<html>
+  <head>
+    <title>Daily.co Translator Demo</title>
+    <script src="https://unpkg.com/@daily-co/daily-js"></script>
+    <style>
+      body {
+        font-family: -apple-system, BlinkMacSystemFont, avenir next, avenir,
+          segoe ui, helvetica neue, helvetica, Cantarell, Ubuntu, roboto, noto,
+          arial, sans-serif;
+        font-size: 1.6em;
+      }
+      button {
+        font-size: 1em;
+      }
+      select {
+        font-size: 1em;
+      }
+      div#controls-left select {
+        font-size: 0.6em;
+      }
+
+      div#videos {
+        display: flex;
+        width: 100%;
+        max-width: 100%;
+        height: 90%;
+        display: flex;
+      }
+      div#videos > div {
+        padding: 1em;
+        flex-grow: 1;
+        display: flex;
+        flex-direction: column;
+      }
+      div#videos > div video {
+        width: 100%;
+        height: 100%;
+        object-fit: cover;
+      }
+      div#videos .subtitles {
+        height: 5em;
+      }
+      div#videos .subtitles p {
+        text-align: center;
+        margin-block-start: 0.2em;
+      }
+      div#controls {
+        padding: 0.2em;
+        display: flex;
+        justify-content: space-between;
+      }
+    </style>
+  </head>
+  <body onload="main()">
+    <div id="controls">
+      <div id="controls-left">
+        <button id="load-devices" onclick="loadDevices()" style="display: none">
+          Load Cameras
+        </button>
+        <select id="cams" style="display: none"></select>
+        <select id="mics" style="display: none"></select>
+        <button id="join" onclick="joinRoom()">Join Room</button>
+        <button id="leave" onclick="leaveRoom()">Leave Room</button>
+      </div>
+      <div id="controls-right">
+        <span>Language:</span>
+        <select id="language" onChange="changeLanguage()">
+          <option value="english" selected>English</option>
+        </select>
+      </div>
+    </div>
+    <div id="videos"></div>
+    <div id="audios"></div>
+    <script>
+      const urlParams = new URLSearchParams(window.location.search);
+
+      const languageSelect = document.getElementById("language");
+      const loadButton = document.querySelector("button#load-devices");
+      const camSelect = document.querySelector("select#cams");
+      const micSelect = document.querySelector("select#mics");
+      const host = !!urlParams.get("host");
+
+      function cap(string) {
```

Comment:
Suggest calling this parameter something other than "string" as it is commonly used as a built-in type name in other languages and can be confusing to read.